packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -26,77 +32,83 @@ export function getLatestNodeVersion(): NodeVersion {
 
 export async function getSupportedNodeVersion(
   engineRange?: string,
-  silent?: boolean
+  isAuto?: boolean
```

Comment:
I renamed this variable because it wasn't clear before.

When `isAuto: true` then we are not using package.json engines and it was assigned automatically from the project or zero config.